StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/MessageSendable.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MessageSendable.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/2/24.
+//
+
+import Foundation
+
+protocol MessageSender {
+    associatedtype MessageType: Codable & Equatable
+    var name: String { get }
+    var message: MessageType { get }
```

Comment:
nits:
- some docstrings would be helpful
- thoughts about `payload` or `parameter` > `message`?
- Only need `Encodable` (instead of `Codable`)
```suggestion
/// Sends a message to the webview by calling a function on `window`
protocol MessageSender {
    associatedtype Payload: Encodable & Equatable
    /// Name of the method (e.g. `updateConnectInstance`)
    var name: String { get }
    /// Function param
    var payload: Payload { get }
```